paddle/fluid/distributed/collective/process_group_gloo.h

```diff
@@ -210,6 +216,11 @@ class ProcessGroupGloo : public ProcessGroupWithoutStream {
     return platform::DeviceContextPool::Instance().Get(place);
   }
 
+  phi::DeviceContext* GetDeviceContext(const Place& place,
```

Comment:
Do not need this method.

Reply:
An exception will be thrown if this method is not used.

Reply:
I add an exception when use_calc_stream is true.